ToyModels/plugins/ParticleGunVarProducer.cc

```diff
@@ -198,8 +198,8 @@ ParticleGunVarProducer<osu::Track,TYPE(muons)>::passesVeto (const osu::Track &pr
 #if DATA_FORMAT == MINI_AOD_2017
   bool passes = probe.deltaRToClosestPFMuon () > 0.15
              && probe.hitAndTOBDrop_bestTrackMissingOuterHits () >= 3.0;
-#elif DATA_FORMAT == MINI_AOD_2022_CUSTOM
-  bool passes = probe.deltaRToClosestPFMuon () > 0.15
+#elif DATA_FORMAT == MINI_AOD_2022_CUSTOM || DATA_FORMAT == MINI_AOD_ONLY_2022_CUSTOM
```

Comment:
remove custom?

Reply:
Both this and comment above should still be `CUSTOM`, because we still use the custom data format for it. It is defined here `https://github.com/OSU-CMS/OSUT3Analysis/pull/258/files#diff-d6e4775b97e4aa524a4d346aefa029480baff80bc6a254fc21b1e8df224b0cd7R22`